reference/resources/payment_method--tokenized.yaml

```
@@ -98,3 +98,20 @@ properties:
       be used for. If this value is `null` the payment method may be
       used in multiple countries.
     nullable: true
+
+  last_replaced_at:
+    type: string
+    nullable: true
+    description: |-
+      The date and time when this payment method was last replaced.
+      This value is always `null` for non-card payment methods.
+    format: date-time
+    example: "2023-07-26T19:23:00.000+00:00"
+
+  has_replacement:
+    type: boolean
+    nullable: false
+    description: |-
+      Whether this payment method has a pending replacement that hasn't been applied yet.
+      This value is always `false` for non-card payment methods.
```

```suggestion
      Whether this card has a pending replacement that hasn't been applied yet.
      This value is always `false` for non-card payment methods.
```

If it's always a card then let's call it a card.

Again, I think this needs a bit more explanation as to what a "replacement" is. E.g. a new set of card details.

---

Done: [Extend descriptions of new fields](https://github.com/gr4vy/gr4vy-openapi/pull/377/commits/eaf2761ecc766374fd277474ed635b71e1c94c36)